src/main/java/com/example/demo/GlobalExceptionHandler.java

```diff
@@ -64,4 +65,12 @@ public ResponseEntity<ErrorResponseBody> handleUserNotCreatedException(UserNotCr
         return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(errorResponseBody);
     }
 
+        @ExceptionHandler(PostNotDeletedException.class)
```

Comment:
インデントが不揃いなのが気になりました。
```suggestion
    @ExceptionHandler(PostNotDeletedException.class)
```